tfawserr/awserr_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package tfawserr
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/service/sts/types"
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
This should be `github.com/aws/aws-sdk-go-v2/aws`.

Reply:
Copy and paste :(